aspnetcore/signalr/javascript-client.md

```diff
@@ -261,9 +261,47 @@ The following code demonstrates a typical manual reconnection approach:
 1. A function (in this case, the `start` function) is created to start the connection.
 1. Call the `start` function in the connection's `onclose` event handler.
 
-[!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=30-40)]
+[!code-javascript[](javascript-client/samples/3.x/SignalRChat/wwwroot/chat.js?range=30-46)]
 
-A real-world implementation would use an exponential back-off or retry a specified number of times before giving up.
+A production implementation would use an exponential back-off or retry a specified number of times before giving up.
+
```

Comment:
```suggestion
<a name="bsleep"></a>

```

Reply:
Do the docs also support comments `// this is used by code in SignalR, don't rename or remove`?

Reply:
As HTML ... `<!-- blah blah blah -->`

Reply:
... the bookmark thing is under ... as Ryan would say ... "civil debate" 😆 lol ... `name` is depreciated under HTML5, and that's our doctype.

Reply:
Like real code, people skip comments, that's why I like the shorter permanent link. We can add both.

Reply:
Ah of course, html comments 👍 
```suggestion
<!-- This anchor is used by code in the SignalR Typescript client, do not rename or remove without considering the impacts there -->
<a name="bsleep"></a>

```

Reply:
The following is valid ...

```html
<h2 id="bsleep">Browser sleeping tab</h2>
```

... but markdown linting isn't good for it. ....... and docs management might send a hit squad to your house! 😨 (but I had to do it in a couple of pivoting docs to avoid broken bookmarks)